tools/toolchains/arm.py

```diff
@@ -456,7 +474,10 @@ def __init__(self, target, *args, **kwargs):
         self.elf2bin = join(TOOLCHAIN_PATHS["ARMC6"], "fromelf")
 
     def _get_toolchain_labels(self):
-        return ["ARM", "ARM_STD", "ARMC6"]
+        if getattr(self.target, "default_toolchain", "ARM") == "uARM":
+            return ["ARM", "ARM_MICRO"]
+        else:
+            return ["ARM", "ARM_STD"]
```

Comment:
This used to have an `ARMC6` entry as well. Please add that back.

Reply:
@theotherjimmy - Please see the latest commit, I have the 'ARMC6' label issue fixed.